### PR TITLE
Fixed some typos in workshop slides.

### LINKED ISSUE #27796: Moose Workshop: change heat capacity to density * specific heat
### Motivation

The Porous flow example that is the foundation of the MOOSE workshop training formulates the energy conservation equation using heat capacity instead of the more common formulation that uses specific heat and density. The workshop slides should be updated to use the more common formulation, which is what is actually implemented in MOOSE.

### Design
Update workshop slides, replace all occurrences of "C" with "\rho c_p". 

### Impact

Improve consistency between the tutorial problem statement and its implementation in MOOSE, avoiding confusion.

### PR DESCRIPTION
**Pull request by pbehne**
Fixed some typos in workshop slides.

Closes #27796.

### CODE REVIEW

File: tutorials/darcy_thermo_mech/doc/content/workshop/systems/outputs.md [hunk 1]
```diff
@@ -88,8 +90,6 @@ using the short-cut syntax.  sub-blocks use the actual sub-block name as the suf
 []
 ```
 
-The use of 'file_base' anywhere in the `[Outputs]` block disables all default naming behavior.
```
Comment (pbehne) [inline tutorials/darcy_thermo_mech/doc/content/workshop/systems/outputs.md:91]: I tested this. This statement is false. The default naming behavior is only disabled in the block where `file_base` is specified.

File: tutorials/darcy_thermo_mech/step03_darcy_material/include/kernels/DarcyPressure.h [hunk 2]
```diff
@@ -31,7 +31,7 @@ class DarcyPressure : public ADKernel
   /// The permeability. Note that this is declared as a \p MaterialProperty. This means that if
   /// calculation of this property in the producing \p Material depends on non-linear variables, the
   /// derivative information will be lost here in the consumer and the non-linear solve will suffer
-  const ADMaterialProperty<Real> & _permeability;
+  const MaterialProperty<Real> & _permeability;
```
Comment (pbehne) [inline tutorials/darcy_thermo_mech/step03_darcy_material/include/kernels/DarcyPressure.h:34]: Changing this to `MaterialProperty` to fit the above descriptive comment. If this change is undesirable, the comment needs to be updated.

**Comment by pbehne** [comment]
@cticenhour

**Comment by pbehne** [comment]
@permcody, need to look into "SetupDarcyAction" on slide 59.3.

**Comment by pbehne** [comment]
Update checkpoint slides to talk about wall-time checkpoints.

**Comment by pbehne** [comment]
> @permcody, need to look into "SetupDarcyAction" on slide 59.3.

Done.

**Comment by pbehne** [comment]
> Update checkpoint slides to talk about wall-time checkpoints.

Done.

**Comment by pbehne** [comment]
@MengnanLi91, @grmnptr, @loganharbour, @cticenhour, @permcody: feel free to push any updates/fixes to the training material. I think I am done.

**Comment by pbehne** [comment]
Actually, I forgot to make the heat capacity/specific heat/density consistent throughout the tutorial. I'll work on that.

**Comment by pbehne** [comment]
> Actually, I forgot to make the heat capacity/specific heat/density consistent throughout the tutorial. I'll work on that.

Done.

File: tutorials/darcy_thermo_mech/doc/content/workshop/problem/overview.md [hunk 3]
```diff
@@ -45,17 +45,17 @@ $p$ and $T$:
 -\nabla \cdot \frac{\mathbf{K}}{\mu} \nabla p  = 0
 
 !equation
-C\left( \frac{\partial T}{\partial t} + \epsilon \vec{u}\cdot\nabla T \right) - \nabla \cdot k \nabla T = 0
+\rho c_p \left( \frac{\partial T}{\partial t} + \epsilon \vec{u}\cdot\nabla T \right) - \nabla \cdot k \nabla T = 0
 
 !---
 
-The parameters $\rho$, $C$, and $k$ are the porosity-dependent density, heat capacity, and thermal
+The parameters $\rho$, $c_p$, and $k$ are the porosity-dependent density, heat capacity, and thermal
```
Comment (GiudGiud) [inline tutorials/darcy_thermo_mech/doc/content/workshop/problem/overview.md:52]:
```suggestion
The parameters $\rho$, $c_p$, and $k$ are the porosity-dependent density, specific heat capacity, and thermal
```

**Review by GiudGiud — COMMENTED** [review]
looks good to me

**Comment by GiudGiud** [comment]
@pbehne your thoughts on the review?

**Comment by pbehne** [comment]
> @pbehne your thoughts on the review?

Sorry, I only saw your "looks good" comment, not your suggestion. Good catch.

**Comment by GiudGiud** [comment]
please squash it with the previous commits

**Review by GiudGiud — APPROVED** [review]
lots of good catches. thanks for the care

**Pull request merged**